receiver/jaeger/trace_receiver.go

```diff
@@ -47,6 +49,7 @@ import (
 type Configuration struct {
 	CollectorThriftPort int `yaml:"tchannel_port"`
 	CollectorHTTPPort   int `yaml:"collector_http_port"`
+	CollectorGRPCPort   int `yaml:"collector_http_port"`
```

Comment:
Conflict in the YAML tag with `collector_http_port`, please make this `collector_grpc_port`